_posts/2016/01/2016-01-28-instructor-debriefing-round-02.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+authors: ["Rayna Harris", "Christina Koch", "Bill Mills", "Raniere Silva"]
+title: "2016 Post-Workshop Instructor Debriefing, Round 02"
+date: 2016-01-28
+time: "13:00:00"
+category: ["Debriefing"]
+---
+<!-- start excerpt -->
+On January 26, we ran the 2nd round of Post-Workshop Instructor Debriefing Sessions. Rayna and Christina hosted the morning debriefings while Bill and Raniere hosted the evening debriefing.  A topic of interest today was on installations. Read on for details. 
+<!-- end excerpt -->
+
+## Thoughts on the "check installation" scripts
+There is in an ongoing discussion about whether or not the **"check installation" scripts** for workshop pages should be updated. So, we asked the attendees how often these were being using and how useful they were. Some comments we received were:
+1. only a handful of students use them before coming to the workshop
+2. instructors/helpers use them when diagnosing install problems on site
+3. some cleaning of the script would be useful to remove checks for outdated and/or irrelevant tools
+
+If you want to contribute to your two cents, the you can submit an issue [here](https://github.com/swcarpentry/windows-installer).
+
+##Thoughts on dealing with installation problems
+We know that poor planning, sub-par wifi, lack of admin privileges, improper installs, power outages, and a bunch of other things can give rise to installation problems for a learner. 
+
```

Comment:
Add: *A number of groups in Australia have used a cloud-based service for providing learners with homogeneous running environments; this solution has performed well a number of times, but is subject to server and network outages as was the case in the Brisbane workshop discussed in the evening session.*